packages/three-vrm-springbone/src/tests/VRMSpringBoneColliderShapeCapsule.test.ts

```diff
@@ -87,19 +87,19 @@ describe('VRMSpringBoneColliderShapeCapsule', () => {
       });
 
       it('must calculate a collision properly, object is near the tail', () => {
-        const colliderMatrix = new THREE.Matrix4();
+        const colliderMatrix = new THREE.Matrix4().makeTranslation(shape.offset);
         const objectPosition = new THREE.Vector3(3.0, 0.0, 0.0);
-        const objectRadius = 1.0;
+        const objectRadius = 2.0;
```

Comment:
what is the intention of the change of `objectRadius` ?

Reply:
The test was the only one that didn't report a collision (`dist > 0`). The output direction (`dir`) is now only normalized when there is a collision. So either the assertion on `dir` could be removed or the test changed to have a collision.

Changing the `objectRadius` seemed the easiest way to cause a collision to take place. Especially since the test is intended to test the tail of a capsule, it feels incomplete if it only checked the `dist`.

Reply:
I see. Thank you for the explanation!